themes/default/content/docs/intro/concepts/secrets.md

```diff
@@ -482,6 +482,14 @@ $ pulumi stack init my-stack \
 
 If you have previously configured the AWS CLI, the same credentials will be used. These can also be overridden using the standard `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables. For more options, refer to the [AWS Go SDK documentation](https://docs.aws.amazon.com/sdk-for-go/api/aws/session/).
 
+{{% notes "info" %}}
+As of Pulumi CLI vX.X.X, instead of specifying the AWS Profile using the `AWS_PROFILE` environment variable, you can append the profile as a querystring to the above KMS identifiers:
```

Comment:
```suggestion
As of Pulumi CLI v3.33.1, instead of specifying the AWS Profile using the `AWS_PROFILE` environment variable, you can append the profile as a querystring to the above KMS identifiers:
```